hive.go

```diff
@@ -466,6 +478,17 @@ func (c *Cursor) Execute(ctx context.Context, query string, async bool) {
 			}
 			return
 		}
+
+		// Flush logs after execution is finished
+		if c.Logs != nil {
+			logs := c.FetchLogs()
+			if c.Error() != nil {
+				c.state = _ERROR
```

Comment:
Want to flag this line - what state should this final failure to send the logs indicate? `_ERROR`? or `_ASYNC_ENDED`

Reply:
Let's leave it to _ASYNC_ENDED